tests/test_oml/test_postprocessor/test_pairwise_embeddings.py

```diff
@@ -138,46 +146,6 @@ def test_trivial_processing_fixes_broken_perfect_case() -> None:
             assert metric_upd >= metric, (key, metric, metric_upd)
 
 
-class DummyPairwise(IPairwiseModel):
-    def __init__(self, distances_to_return: Tensor):
-        super(DummyPairwise, self).__init__()
-        self.distances_to_return = distances_to_return
-        self.parameter = torch.nn.Linear(1, 1)
-
-    def forward(self, x1: Tensor, x2: Tensor) -> Tensor:
-        return self.distances_to_return
-
-    def predict(self, x1: Tensor, x2: Tensor) -> Tensor:
-        return self.distances_to_return
-
-
-@pytest.mark.long
```

Comment:
I removed this test because it was too tricky and it's hard to support when changing interfaces.